### PR TITLE
Add link to user survey

### PR DESCRIPTION
**Pull request by TomAugspurger**
Is this a good idea?

Screen Shot 2019-06-29 at 12 20 47 PM

@jrbourbeau could you run this by your special consultant for feedback on the best place to put it?

### CODE REVIEW

**Comment by TomAugspurger** [comment]
One downside, this will always be there when you reload the page. I'm guessing we could use a cookie to check if you've dismissed it.

File: index.html [hunk 1]
```diff
@@ -95,6 +95,14 @@
 
       <div class="container marketing p-md-5">
 
+
+        <div class="alert alert-warning alert-dismissible fade show" role="alert">
```
Comment (jrbourbeau) [inline index.html]: I also received some styling suggestions, which we are free to take or leave

```suggestion
        <div class="alert alert-warning alert-dismissible fade show" role="alert" style="text-align: center;font-size: 1.1em;max-width: 720px;margin-left: auto;margin-right: auto;">
```

File: index.html [hunk 2]
```diff
@@ -95,6 +95,14 @@
...
+            Take the <a href="https://t.co/OGrIjTLC2G">2019 Dask User Survey.</a>
```
Comment (jrbourbeau) [inline index.html]:
```suggestion
            Help inform future development and improve the Dask community experience - take the <a href="https://t.co/OGrIjTLC2G">2019 Dask User Survey</a>
```

**Review by jrbourbeau — COMMENTED** [review]
Good idea @TomAugspurger, I'm +1 for adding this

I've also been informed that the current location is a good place to put it

**Comment by jrbourbeau** [comment]
Including the above comments gives: 

Screen Shot 2019-06-29 at 8 29 45 PM

**Review by jrbourbeau — APPROVED** [review]

**Comment by jrbourbeau** [comment]
Thanks @TomAugspurger

**Pull request merged**